frontend/src/routes/Routes.jsx

```diff
@@ -1,3 +1,23 @@
-export const Routes = () => {
-  return <div>Routes</div>;
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Landing } from "../pages/Landing.jsx";
+import { Login } from "../pages/Login.jsx";
+import { Register } from "../pages/Register.jsx";
+import { Dashboard } from "../pages/Dashboard.jsx";
+import { Logout } from "../pages/Logout.jsx";
+
+
+export const AppRoutes = () => {
+  return <>
+    <Router>
```

Comment:
I would have done more or less the same. 😉

Reply:
great minds think alike 😄